def/pl/menu_win32.cson

```diff
@@ -0,0 +1,307 @@
+          "Run &Benchmarks":
+            value: "Uruchom benchmark(&B)"
+          "Toggle Developer &Tools":
+            value: "Narzędzia programistyczne(&T)"
+      "&Increase Font Size":
+        value: "Zwiększ rozmiar czcionki(&I)"
+      "&Decrease Font Size":
+        value: "Zmniejsz rozmiar czcionki(&D)"
+      "Re&set Font Size":
+        value: "Zresetuj rozmiar czcionki(&S)"
+      "Toggle Soft &Wrap":
+        value: "Przełącz na miękkie zawijanie(&W)"
+      "Toggle Command Palette":
+        value: "Przełącz na linię poleceń"
+      "Toggle Tree View":
+        value: "Przełącz na widok drzewa"
+  "&Selection":
+    value: "Zaznaczenia(&S)"
+    submenu:
+      "Add Selection &Above":
+        value: "Dodaj zaznaczenie powyżej(&A)"
+      "Add Selection &Below":
+        value: "Dodaj zaznaczenie poniżej(&B)"
+      "S&plit into Lines":
+        value: "Podziel na linie(&P)"
+      "Single Selection":
+        value: "Pojedyncze zaznaczenie"
+      "Select to &Top":
+        value: "Zaznacz do góry(&T)"
+      "Select to Botto&m":
+        value: "Zaznacz do dołu(&M)"
+      "Select &Line":
+        value: "Zaznacz linię(&L)"
+      "Select &Word":
+        value: "Zaznacz słowo(&W)"
+      "Select to Beginning of W&ord":
+        value: "Zaznacz do początku słowa(&O)"
+      "Select to Beginning of L&ine":
+        value: "Zaznacz do początku linii(&I)"
+      "Select to First &Character of Line":
+        value: "Zaznacz do pierwszego znaku w linii(&C)"
+      "Select to End of Wor&d":
+        value: "Zaznacz do końca słowa(&D)"
+      "Select to End of Lin&e":
+        value: "Zaznacz do końca linii(&E)"
+      "Select Inside Brackets":
+        value: "Zaznacz wewnątrz nawiasów"
+  "F&ind":
+    value: "Szukaj(&I)"
+    submenu:
+      "Find in Buffer":
+        value: "Szukaj w buforze"
+      "Replace in Buffer":
+        value: "Zmień w buforze"
+      "Select Next":
+        value: "Zaznacz następny"
+      "Select All":
+        value: "Zaznacz wszystko"
+      "Toggle Find in Buffer":
+        value: "Przełącz na szukanie w buforze"
+      "Find in Project":
+        value: "Szukaj w projekcie"
+      "Toggle Find in Project":
+        value: "Przełącz na szukanie w projekcie"
+      "Find All":
+        value: "Szukaj wszędzie"
+      "Find Next":
+        value: "Znajdź następny"
+      "Find Previous":
+        value: "Znajdź poprzedni"
+      "Replace Next":
+        value: "Zastąp następny"
+      "Replace All":
+        value: "Zastąp wszystko"
+      "Clear History":
+        value: "Wyczyść historię"
+      "Find Buffer":
+        value: "Przeszukaj bufor"
+      "Find File":
+        value: "Szukaj pliku"
+      "Find Modified File":
+        value: "Szukaj modyfikowanego pliku"
+  "&Packages":
+    value: "Pakiety(&P)"
+  "&Help":
+    value: "Pomoc(&H)"
+    submenu:
+      "View &Terms of Use":
+        value: "Pokaż warunki korzystania"
+      "View &License":
+        value: "Pokaż licencję"

```

Comment:
Please add *hotkey hint prompt* `&` here. 
You may select whichever letter you feel comfortable with.